sdk/attestation/azure-security-attestation/samples/README.md

```diff
@@ -4,7 +4,6 @@ languages:
     - python
 products:
     - azure
-    - azure-security-attestation
```

Comment:
It exists as "azure-attestation":
https://github.com/azure/azure-sdk-for-java/tree/main/sdk/attestation/azure-security-attestation/src/samples/java/com/azure/security/attestation

And it works under that name here:
https://docs.microsoft.com/en-us/samples/browse/?products=azure-attestation&expanded=azure

![image](https://user-images.githubusercontent.com/1050156/179570064-9fefa5b5-7b12-4a77-a85b-75aaed49d276.png)